app/controllers/organen/beheer/index.js

```diff
@@ -1,18 +1,101 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
-import { tracked } from '@glimmer/tracking';
+import moment from 'moment';
 
 export default class OrganenBeheerIndexController extends Controller {
   @service router;
+  @service store;
 
-  @tracked filter = '';
-  @tracked page = 0;
-  sort = 'naam';
-  size = 5;
+  active_sort = 'naam';
+  inactive_sort = 'naam';
 
   @action
   createNewOrgaan() {
     this.router.transitionTo('organen.beheer.new');
   }
+
+  @action
+  async getNbMembers(orgaan) {
```

Comment:
could this be a method on the bestuursorgaan model?

Reply:
Not sure if that is possible, might be difficult with the async, but I could try if you want.